components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -288,6 +292,12 @@ export default class CollectionItem extends mixins(
       })
     }
 
+    if (this.searchQuery.owned) {
+      params.push({
+        currentOwner: { equalTo: this.accountId },
+      })
+    }
```

Comment:
This will look into currentOwner in `CollectionEntity`

Reply:
No, the `currentOwner` and also other search params are for `NFTEntity`

Reply:
I have not been able to display the nfts I own from a collection although I see them in my profile

Reply:
I will check it later

Reply:
yeah better!